pytest_order/sorter.py

```diff
@@ -224,6 +224,21 @@ def items_from_class_label(self, label: str, item: Item) -> List[Item]:
                         items.append(self.node_ids[node_id])
         return items
 
+    def items_from_path_label(self, label: str, item: Item) -> List[Item]:
+        items = []
+        item_id = item.node_id
+        label_len = len(label)
+        for node_id in self.node_ids:
+            if node_id.count("::") == 2:
+                path_index = node_id.index("::")
+                if label.endswith('/') and label in node_id:  # directory ordering
+                    items.append(self.node_ids[node_id])
+                elif node_id[:path_index].endswith(label):    # end-of-filename ordering
```

Comment:
This implies that there is only one directory ending with `label`, though it could be documented that `label` has to be unambiguous in that respect.

Reply:
I thought it would be easier to require ending the label with a `/` to refer to a directory, which would incidentally avoid confusing a directory and any other kind of path/entity.

Reply:
Agreed, makes sense.